addons/api/tests/unit/models/target-test.js

```diff
@@ -384,4 +384,24 @@ module('Unit | Model | target', function (hooks) {
     const model = store.peekRecord('target', '123abc');
     await model.removeCredentialSource('2');
   });
+
+  test('it has isSSH property and returns the expected values', async function (assert) {
```

Comment:
```suggestion
  test('it has isSSH property and returns the expected values', function (assert) {
```
It does not look like we need this to be async.

Reply:
good catch! updated!